### PR TITLE
Fix substation creation when name is empty or contain only spaces

### PR DESCRIPTION
**Pull request by Walid-Sahnoun**

### CODE REVIEW

**Review by FranckLecuyer — APPROVED** [review]

File: src/utils/rest-api.js [hunk 1]
```diff
@@ -1658,6 +1658,13 @@ export function createTwoWindingsTransformer(
     });
 }
 
+function checkEquipementNameEmptyOrContainsOnlySpaces(name) {
```
Comment (SlimaneAmar) [inline src/utils/rest-api.js:1661]: Remove it

File: src/utils/rest-api.js [hunk 2]
```diff
@@ -1690,7 +1697,8 @@ export function createSubstation(
         },
         body: JSON.stringify({
             equipmentId: substationId,
-            equipmentName: substationName,
+            equipmentName:
+                checkEquipementNameEmptyOrContainsOnlySpaces(substationName),
```
Comment (SlimaneAmar) [inline src/utils/rest-api.js:1701]: substationName.trim() === '' ? null : substationName.trim()

**Review by SlimaneAmar — CHANGES_REQUESTED** [review]

File: src/utils/rest-api.js (hunk 1 above)
Comment (Walid-Sahnoun) [reply src/utils/rest-api.js]: done

File: src/utils/rest-api.js (hunk 2 above)
Comment (Walid-Sahnoun) [reply src/utils/rest-api.js]: done

**Pull request closed without merging**